scripts/testing/mega_comprehensive_model_test.py

```
@@ -39,59 +39,59 @@ def __init__(self, model_path="deployment/models/default"):
             self.model = AutoModelForSequenceClassification.from_pretrained(self.model_path)
-            
+
             if torch.cuda.is_available():
                 self.model = self.model.to('cuda')
                 print("✅ Model moved to GPU")
             else:
                 print("⚠️ CUDA not available, using CPU")
-            
+
             print("✅ Model loaded successfully for mega testing")
             return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)